config/deprecation-workflow.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-undef */
+self.deprecationWorkflow = self.deprecationWorkflow || {};
+self.deprecationWorkflow.config = {
+  workflow: [
+    { handler: "throw", matchId: "ember-global" },
+    {
+      handler: "throw",
```

Comment:
Would it not throw anyways?

Reply:
What do you mean with anyways? i removed the two entries in the follow up pr #151

Reply:
I was asking myself if the two options would not generated an error anyways in the CI. But I guess the second would go silently. But committing the `throw` rules seemed special to me, so just wanted to double check.